plugin-server/src/worker/ingestion/utils.ts

```diff
@@ -61,8 +61,13 @@ export function generateEventDeadLetterQueueMessage(
 // These get displayed under Data Management > Ingestion Warnings
 // These warnings get displayed to end users. Make sure these errors are actionable and useful for them and
 // also update IngestionWarningsView.tsx to display useful context.
-export async function captureIngestionWarning(db: DB, teamId: TeamId, type: string, details: Record<string, any>) {
-    await db.kafkaProducer.queueMessage({
+export async function captureIngestionWarning(
```

Comment:
we were passing in `DB` which blob consumer doesn't have so that we could read the kafkaProducer from it.

if we pass the kafkaProducer directly usage is simpler and the blob consumer doesn't need a new dependency